src/odemis/acq/test/move_test.py

```diff
@@ -35,12 +35,15 @@
 import unittest
 
 logging.getLogger().setLevel(logging.DEBUG)
+logging.basicConfig(format="%(asctime)s  %(levelname)-7s %(module)s:%(lineno)d %(message)s")
+
 
 CONFIG_PATH = os.path.dirname(odemis.__file__) + "/../../install/linux/usr/share/odemis/"
 ENZEL_CONFIG = CONFIG_PATH + "sim/enzel-sim.odm.yaml"
 METEOR_CONFIG = CONFIG_PATH + "sim/meteor-sim.odm.yaml"
+MIMAS_CONFIG = CONFIG_PATH + "sim/mimas-sim.odm.yaml"
 
-
+@unittest.skip #DEBUG
```

Comment:
why do you have this here?

Reply:
I have this comment to remember removing it before the PR :sweat_smile: